### ISSUE TITLE
README is incomplete.

### ISSUE THREAD

**Issue by JimmyMcBride** [michelangelo17/michelangelo.codes#10]
Absolutely love your code and your use of all those emotional things. I think your flex naming conventions were particularly clever (aic fdc). Your app.js is also super beautiful and clean. Just a stack of components

One thing I've discovered with styled-components is that you can use the `as` to change the root element of that component. So I have a component named `Text` that looks like:

```javascript
const Text = div`
  font: // value;
  font-weight: // value;
  color: // value;
  // etc...
`
```

Then when I use it I change div to the text element I want. `<Text as="h1" xlf bold>I'm an h1!</Text>`.
The `as` property changes it from div to h1.

Also, where's your readme???? The code looks fantastic, but I wanna see the actual site now! Lol

**Comment by michelangelo17** [comment]
Yeah, I actually do use the 'as' feature often, even on this site, don't know why it didn't occur to me to consolidate all those custom tags which basically do the same thing. That is definitely going to happen. As far as a readme, it's just my project site so it didn't seem necessary, I'm not linking to the github for it. If you want to see it though, the web address literally is michelangelo.codes :-)
